pkgs/top-level/all-packages.nix

```diff
@@ -435,6 +435,8 @@ in
 
   _0x0 = callPackage ../tools/misc/0x0 { };
 
+  _3llo = callPackage ../applications/misc/3llo { };
```

Comment:
Would this be a better fit for `tools` since it's such a small program?

##### also note this is already in the `### TOOLS` section and not `### APPLICATIONS`